faq/sql-faq.md

```diff
@@ -285,6 +307,17 @@ Generally the input data of `root task` comes from `cop task`; when `root task`
 
 See [The TiDB Command Options](/command-line-flags-for-tidb-configuration.md).
 
+### How to avoid hotspot issues and achieve load balancing? Is hot partition or range an issue in TiDB?
+
+To learn the scenarios that cause hotspots, refer to [common hotpots](/troubleshoot-hot-spot-issues.md#common-hotspots). To resolve hotspot issues, you can use the following built-in TiDB features:
```

Comment:
The solutions of hotspot issues depends on the various causes.  It's not guaranteed the given solutions resolve all problems.  So we can say TiDB provides a few built-in features to resolve hotspot issues.. instead of saying the issues have to be resolved by these built-in features.

Reply:
addressed